diff -ruN a/rust/common/src/util/os_utils.rs b/rust/common/src/util/os_utils.rs
--- a/rust/common/src/util/os_utils.rs	2026-10-09 16:25:19.695107392 +0000
+++ b/rust/common/src/util/os_utils.rs	2026-10-09 16:25:19.725937372 +0000
@@ -16,7 +16,7 @@
 //! # os utility functions
 
 use std::ffi::CString;
-#[cfg(any(target_os = "linux", target_os = "macos"))]
+#[cfg(unix)]
 use std::os::fd::AsRawFd;
 #[cfg(target_os = "windows")]
 use std::os::windows::io::AsRawHandle;
@@ -36,15 +36,15 @@
     subdir: &'static str,
 }
 
-#[cfg(target_os = "linux")]
+#[cfg(all(unix, not(target_os = "macos")))]
 const DISCOVERY: DiscoveryDir = DiscoveryDir { root_env: "XDG_RUNTIME_DIR", subdir: "" };
 #[cfg(target_os = "macos")]
 const DISCOVERY: DiscoveryDir =
     DiscoveryDir { root_env: "HOME", subdir: "Library/Caches/TemporaryItems" };
 #[cfg(target_os = "windows")]
 const DISCOVERY: DiscoveryDir = DiscoveryDir { root_env: "LOCALAPPDATA", subdir: "Temp" };
-#[cfg(not(any(target_os = "linux", target_os = "macos", target_os = "windows")))]
-compile_error!("netsim only supports linux, Mac, and Windows");
+#[cfg(not(any(unix, target_os = "windows")))]
+compile_error!("netsim only supports Unix-like systems and Windows");
 
 /// Get discovery directory for netsim
 pub fn get_discovery_directory() -> PathBuf {
@@ -177,7 +177,7 @@
     let mode_c = CString::new("w")?;
 
     // Obtain the raw file descriptors for stdout.
-    #[cfg(any(target_os = "linux", target_os = "macos"))]
+    #[cfg(unix)]
     let stdout_fd = std::io::stdout().as_raw_fd();
     #[cfg(target_os = "windows")]
     // SAFETY: This operation allows opening a runtime file descriptor in Windows.
@@ -186,7 +186,7 @@
         unsafe { libc::open_osfhandle(std::io::stdout().as_raw_handle() as isize, libc::O_RDWR) };
 
     // Obtain the raw file descriptors for stderr.
-    #[cfg(any(target_os = "linux", target_os = "macos"))]
+    #[cfg(unix)]
     let stderr_fd = std::io::stderr().as_raw_fd();
     #[cfg(target_os = "windows")]
     // SAFETY: This operation allows opening a runtime file descriptor in Windows.
diff -ruN a/rust/hostapd-rs/src/hostapd_sys/mod.rs b/rust/hostapd-rs/src/hostapd_sys/mod.rs
--- a/rust/hostapd-rs/src/hostapd_sys/mod.rs	2026-10-09 16:25:19.704145894 +0000
+++ b/rust/hostapd-rs/src/hostapd_sys/mod.rs	2026-10-09 16:25:19.726045824 +0000
@@ -77,7 +77,7 @@
 //! # fn main() {}
 //! ```
 
-#[cfg(target_os = "linux")]
+#[cfg(all(unix, not(target_os = "macos")))]
 include!("linux/bindings.rs");
 
 #[cfg(target_os = "macos")]
diff -ruN a/src/util/os_utils.cc b/src/util/os_utils.cc
--- a/src/util/os_utils.cc	2026-10-09 16:25:19.710793824 +0000
+++ b/src/util/os_utils.cc	2026-10-09 16:25:19.726120566 +0000
@@ -43,10 +43,10 @@
 DiscoveryDir discovery{
 #if defined(_WIN32)
     "LOCALAPPDATA", "Temp"
-#elif defined(__linux__)
-    "XDG_RUNTIME_DIR", ""
 #elif defined(__APPLE__)
     "HOME", "Library/Caches/TemporaryItems"
+#elif defined(__unix__)
+    "XDG_RUNTIME_DIR", ""
 #else
 #error This platform is not supported.
 #endif
